packages/create-rainbowkit/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # @rainbow-me/create-rainbowkit
 
+## 0.0.4
```

Comment:
I'm not sure what happened, but it looks like your PR is picking up unrelated changes that are already in main.